### DISCUSSION TITLE
Ready to Create Release 2.0.4

### DISCUSSION THREAD

**Discussion by Kevin-Andrew** [IonxSolutions/serilog-sinks-syslog#29] [General]
@colin-anderson, I have merged the two PRs, #26 and #27, which address Issues #11, #20, #25, and #28. There are currently no more pending PRs and these changes do include one bug fix. I'd say you can consider tagging and create a new release from the master branch whenever you have time.

**Comment by colin-anderson** [comment]
Done, [v2.0.4](https://www.nuget.org/packages/Serilog.Sinks.SyslogMessages/2.0.4) is released on NuGet!
